.deployment/docker-compose-overrides.yml

```diff
@@ -1,8 +1,24 @@
 services:
   server:
-    entrypoint: /.deployment/dev-entrypoint.sh
+    entrypoint: /.deployment/dev-entrypoint.sh 8888
```

Comment:
I would change the ports that are published - not the port used internally (ie, `8889:8888`).

Reply:
Ok. I was confused about this. I will change it.

Reply:
This has been changed to use default port.

Reply:
Why not just omit the extra (unused) code used to allow customization of the default port?

Reply:
Yeah. That has been ommitted as well